packages/nimble-components/src/combobox/index.ts

```diff
@@ -575,6 +575,7 @@ export class Combobox
             }
 
             super.selectedIndexChanged(prev, pinnedSelectedIndex);
+            this.setSelectedOptions(); // the base class doesn't call this when no option is selected, but we need to
```

Comment:
Is it worth describing the "why" of the "but we need to" part? I'm assuming this is because of the scenario where we end up with only a disabled option in the dropdown while typing?

Reply:
Actually it's because a number of properties won't get updated otherwise. I've expanded the comment to explain.